src/pybind/mgr/dashboard/frontend/src/app/shared/components/motd/motd.component.html

```diff
@@ -1,8 +1,8 @@
-<cd-alert-panel *ngIf="motd"
-                size="slim"
-                [showTitle]="false"
-                [type]="motd.severity"
-                [dismissible]="motd.severity !== 'danger'"
-                (dismissed)="onDismissed()">
-  <span [innerHTML]="motd.message | sanitizeHtml"></span>
-</cd-alert-panel>
+<ngb-alert *ngIf="motd"
+           class="no-margin-bottom"
+           size="slim"
+           [type]="motd.severity"
+           [dismissible]="motd.severity !== 'danger'"
+           (dismissed)="onDismissed()">
+  <a [innerHTML]="motd.message | sanitizeHtml"></a>
+</ngb-alert>
```

Comment:
@Waadkh7 with this change we have lost the icons used for warning/info/error. If you want to extend the interface of this component you can add, for example, a new `[margin]="false"` input to the cd-alert-panel.

Reply:
Yes i thought it would be better without the icon, just to follow the pattern of Telemetry and password expire notifications.
for that i've take it off. but in case we decided to keep it as it is i can put it back.
For the scroll bar im still trying to figure it out.